pyLibrary/queries/expressions.py

```diff
@@ -2633,9 +2633,11 @@ def to_python(self, not_null=False, boolean=False, many=False):
     def to_sql(self, schema, not_null=False, boolean=False):
         if not isinstance(self.superset, Literal):
             Log.error("Not supported")
-        var = self.value.to_sql(schema)
-        return " OR ".join("(" + var + "==" + sql_quote(v) + ")" for v in json2value(self.superset))
-
+        var = self.value.to_sql(schema,boolean=boolean)
+        if json2value(self.superset):
```

Comment:
I made few changes. Now it returns document with "b" when boolean is True only. This way it won't effect the working of the function if not called for where_clause.